upp-delivery-provisioner/DEVELOPER_README.md

```diff
@@ -86,6 +86,9 @@ export ENVIRONMENT_TAG=
 ## For TEST: t
 export ENVIRONMENT_TYPE=
 
+## the public dns name for the cluster
+export DNS_ADDRESS=
```

Comment:
Do we have an example of `DNS_ADDRESS` that can go in the README?

Or, even better - can we just generate this automatically? All the DNS records should be `${CLUSTER_NAME}-up.ft.com`, so there's no need to get a user to set this themselves.

Reply:
Probably I should have named this better. What I need to have here is the dns name above the regions, if we have more then one. we will decide if the cluster is actually serving cluster or not based on this value.

Reply:
Ah, ok - so this could be `prod-coco-up-read.ft.com` for Prod UK and US, for example?

Popping a couple of examples in the readme so that people know what's expected should be fine then. 👍